bindings/src/test/java/io/pravega/storage/filesystem/FileSystemWrapperTests.java

```diff
@@ -266,6 +266,8 @@ public void validateSetPermissionsAfter() throws IOException {
         Assert.assertTrue(fw.isWritable(filePath));
     }
 
+    // Added to debug issue where file was reported as writable even after making it read only.
```

Comment:
May be this comment could be rewritten. It's not just for debugging, it is fixing an issue. 
also please use "doc comments." like below. Just describe what this method does. Checks whether current user is root. 
```
/**
*/ 
```

And then where it is used in test methods add comments like
`all files are always writeable for root, thus making test irrelevant`  . or something like that

Reply:
Yes. Sure thing. I'll have a look at it and fix it as mentioned above.